From a75fda9a1a7d635d1714df52068cc9fb7988e2c7 Mon Sep 17 00:00:00 2001
From: Yangbo Lu <yangbo.lu@nxp.com>
Date: Fri, 3 Jun 2016 11:40:09 +0800
Subject: [PATCH 48/93] armv8: ls1012a: enable sdhc2 support

A previous patch shifting board specific configurations missed
the sdhc2 support. This patch is to fix it.

Fixes: df5f76d71e30 ("Shift board specific configurations")
Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
---
 include/configs/ls1012aqds.h |    2 ++
 include/configs/ls1012ardb.h |    2 ++
 2 files changed, 4 insertions(+)

diff --git a/include/configs/ls1012aqds.h b/include/configs/ls1012aqds.h
index 1fa7b6f..cc8e0fc 100644
--- a/include/configs/ls1012aqds.h
+++ b/include/configs/ls1012aqds.h
@@ -30,6 +30,8 @@
 #ifdef CONFIG_MMC
 #define CONFIG_CMD_MMC
 #define CONFIG_FSL_ESDHC
+#define CONFIG_FSL_ESDHC_TWO_CONTROLLERS_SUPPORT
+#define CONFIG_FSL_ESDHC_1_NON_REMOVABLE_CARD
 #define CONFIG_SYS_FSL_MMC_HAS_CAPBLT_VS33
 #define CONFIG_GENERIC_MMC
 #define CONFIG_CMD_FAT
diff --git a/include/configs/ls1012ardb.h b/include/configs/ls1012ardb.h
index 23722e0..2eee84a 100644
--- a/include/configs/ls1012ardb.h
+++ b/include/configs/ls1012ardb.h
@@ -64,6 +64,8 @@
 #ifdef CONFIG_MMC
 #define CONFIG_CMD_MMC
 #define CONFIG_FSL_ESDHC
+#define CONFIG_FSL_ESDHC_TWO_CONTROLLERS_SUPPORT
+#define CONFIG_FSL_ESDHC_1_NON_REMOVABLE_CARD
 #define CONFIG_SYS_FSL_MMC_HAS_CAPBLT_VS33
 #define CONFIG_GENERIC_MMC
 #define CONFIG_CMD_FAT
-- 
1.7.9.5

